go.mod

```diff
@@ -29,6 +30,8 @@ require (
 	sigs.k8s.io/controller-runtime v0.12.2
 )
 
+require github.com/pmezard/go-difflib v1.0.0 // indirect
```

Comment:
Add this require block in the indirect dependency (L36) and run make.

Reply:
Solved.